rfcs/rfc-create-ui-architecture.md

```diff
@@ -0,0 +1,306 @@
+# RFC for a "Create UI" Architecture To Support New Filing Types
+
+- Start Date: (2021-03-10)
+- Target Major Version: [Epic #6705](https://github.com/bcgov/entity/issues/6705)
+- Reference Issues: (None)
+- Entity Issue: [Task #6725](https://github.com/bcgov/entity/issues/6725)
+- Implementation PR: (None)
+
+# Summary
+
+Currently, the [Create UI web app](https://github.com/bcgov/bcrs-business-create-ui) allows a user to enter and file a Benefit Company (BCOMP) Incorporation Application (IA).
```

Comment:
bcomp is trademarked, so it shouldn't be used here.

Reply:
I'll remove all instances.

Reply:
done